public/src/scenes/mainTask.js

```diff
@@ -718,38 +727,74 @@ var trialEnd = function () {
     let currentGameState = new GameCache(true, trialNo + 1, maxPressCount, nCoins, coinChoices, randTrialsIdx);
     GameCache.cache = currentGameState;
     EmbedContext.sendMessage('currentGameCache', currentGameState.stringify());
+
+    let isLastTrial = trialNo == nTrials - 1;
+    let camera = this.cameras.main;
+
+    // if the user has been shown the 'Are you still there?' message and they trigger it again, kick them out of the task
+    if (missedTrials >= maxMissedTrials && !isLastTrial && missedTrialDialogShown) {
+        exitGame();
+    }
```

Comment:
Could we also make this configurable in the `versionInfo`? Something like `missedTrialDialogLimit = 1`. If this limit is set to `0` then we never exit the game, and allow infinite 'are you still there?' messages. Otherwise, check how many times we've shows the missed trial dialog to see if we need to exit the game.